vulnerable_people_form/integrations/data/local-restrictions-production.yaml

```diff
@@ -329,7 +329,7 @@ E08000022:
 E08000037:
   name: Gateshead Metropolitan Borough Council
   restrictions:
-    - alert_level: 2
+    - alert_level: 4
```

Comment:
Should we have a tier 4 la in production?

Reply:
this is a mistake - good spot